scratch.js

```diff
@@ -0,0 +1,50 @@
+// new Swipe(slider, {
+//     speed: 600,
+//     auto: 3000,
+//     continuous: true
+//   })
+
```

Comment:
This file should be a copy of the original script.js. New functions were only added to training.js at this time.